js-api-doc/importer.d.ts

```diff
@@ -308,6 +308,37 @@ export interface Importer<sync extends 'sync' | 'async' = 'sync' | 'async'> {
   nonCanonicalScheme?: string | string[];
 }
 
+/**
+ * An opaque branded type, representing the built-in Node Package Importer. This
+ * is used for type checking. Users should use the exposed `nodePackageImporter`
+ * and not instantiate their own.
+ *
+ * @category Importer
+ */
+export type NodePackageImporter = {
+  _NodePackageImporterBrand: unknown;
+};
+/**
+ * The built-in Node Package Importer, which is a Package Importer that resolves
+ * using the standards and conventions of the Node ecosystem. It enables a
+ * `pkg:` URL scheme for usage with `@use` that directs an implementation to
+ * resolve a URL within a dependency.
+ *
+ * A Node Package Importer is exposed as a constant that can be added to the
+ * `importers` option.
+ *
+ * @example
+ *
+ *```js
+ *const sass = require('sass');
+ *sass.compile('style.scss', {
+ *  importers: [sass.nodePackageImporter]
+ *});
```

Comment:
This example should probably use `compileString()` so it can show an actual package import in use (ideally from a package that's already naturally compatible with it).